### ISSUE TITLE
Enhancement: inline docs (M-f12) should display implicit status information

### ISSUE THREAD

**Issue by JasonGross** [cpitclaudel/company-coq#136]
M-f12 is a great replacement for looking up information about constants.  However, I generally want to know which arguments are implicit and which are not, so it'd be useful to display that between the type signature and the definition

**Comment by cpitclaudel** [comment]
Sure. I've been trying to convince the coq devs on and of to do something about this, but I haven't succeeded. Is there a good command that I could use to print this info?

Ideally, I'd like coq to mark the arguments appropriately in definitions themselves, so I could display them in a special font/style.  We already have `{…}` for (maximally inserted?) implicits, so that at least should be easy.

IOW, `Print id` could print `id = fun {A : Type} (x: A) => x` instead of

```
id = fun (A : Type) (x : A) => x

…

Argument A is implicit
```

**Comment by JasonGross** [comment]
Oh, I see, this is just `Print`.  I was thinking you assembled it yourself, and could do `About id | grep '^Arguments'` or something

**Comment by cpitclaudel** [comment]
No, it's not just Print; I do assemble it myself :) It's a combination of About, Print Ltac, Print, and Check.

I was just wondering whether there was a nicer way to get that info :)

**Comment by ejgallego** [comment]
The way implicits work is a bit subtle.

In SerAPI you can query for implicits with `(Query () (Implictits id))` but a bit more work is necessary in order to make them useful I'd guess. Also, the way `Arguments_renaming` interacts with the `implicits_table` seems a bit confusing at first.
